### DISCUSSION TITLE
在威联通Docker 中如何设置Alist的 本机存储 目录

### DISCUSSION THREAD

**Discussion by flyinhere** [AlistGo/alist#1939] [General]
在威联通NAS中，我在一个卷中建立了一个共享文件夹用于Alist的本地网络资料分享，但安装好Alist后，在新建本地存储时，发现不能选择这个共享文件夹，如果选择“/”，则将系统卷的文件和目录全部列出来了，这不是我想要的结果。不知道如何增加共享文件夹啊？

补充说明：
在威联通中，我建立了三个卷，一个系统卷，一个是RAID1阵列卷，一个是静态卷（单独一块硬盘）

**Comment by yh6441** [comment]
![140e80b8410cf2585aa138ae8259cee](https://user-images.githubusercontent.com/21022148/195037789-3146edad-9a4b-41cc-a015-e71c6945a990.jpg)
docker启动加一个挂载点
-v 威联通nas路径:/mnt

**Comment by Yumega** [comment]
docker run -d  --name="alist" 
-v /opt/alist/alist:/opt/alist/data 
 -v /mnt/sdb1/aria2:/myfiles/sdb1/aria2
 -v /mnt/sdb1/Tube:/myfiles/sdb1/Tube
 -v /mnt/sdb1/qbit:/myfiles/sdb1/qbit
-p 5244:5244 
--restart=always
xhofe/alist:latest

然后在根文件夹路径输入 
/myfiles/sdb1/aria2/

就可以读取了

**Reply by rover75** [reply]
感谢大佬，我是Unraid里用Alist，忙活了好几个小时没解决，直到看了您的教程。
